src/telemetry/utils.rs

```diff
@@ -19,28 +19,78 @@ pub async fn telemetry_storage_dirs(cache_dir: &PathBuf) -> (PathBuf, PathBuf) {
 pub fn get_add_del_from_texts(
     text_a: &String,
     text_b: &String,
+    only_one_deletion_allowed: bool,
 ) -> (String, String) {
-    let diff = TextDiff::from_lines(text_a, text_b);
+    let mut text_a_lines = text_a.lines().collect::<Vec<&str>>();
+    let mut text_b_lines = text_b.lines().collect::<Vec<&str>>();
+
+    for s in &mut text_a_lines {
+        *s = s.trim_end().trim_start();
+        // info!("text_a: {}; len: {}", s, s.len());
```

Comment:
if u not sure, use debug!() macros. I will add extra param for debug lvl https://docs.rs/tracing/latest/tracing/macro.debug.html